packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md

```diff
@@ -4,15 +4,25 @@
 
 [Documentation](<TODO>)
 
+This proxy will serve as the backbone for supporting various SPA frameworks by handling server-side rendering (SSR), data queries, and middleware functionalities.
+
 This is a sample setup showing one of how you can configure XM Cloud rendering server.
 
-## Pre-requisites
+## Features Supported
 
-1. SPA sample supports XM Cloud out of the box.
+- Context ID / envoy proxy
+
+- Pages new editing integration via HTTP render engine endpoint (e.g. editing/render)
+
+- Personalization middleware (can do layout manipulation directly since we are always SSR)
+
+- Rewrites middleware
 
-1. Build your SPA app bundle with `jss build`. The build output should be placed in the `dist` folder.
+- Forms support
 
-## Setup
+- Multisite
+
+## Configuration Setup
 
 Open `config.js` and specify your application settings.
```

Comment:
Please expand this section. You can use some comments from the config.ts in ssr-proxy sample here. And you can check our internal doc input for personalize (under 22.3.0 in confluence) for personalizeConfig

Reply:
I guess if the user will open the config.ts then everything is well explained there, I don't find much value in duplicating the same comments again.

Reply:
It would at least be useful to add a few words on what kind of configuration can be done in config.ts